### DISCUSSION TITLE
使用 bb9.0 项目模板新建后没有带解决方案文件 BootstrapBlazorApp.sln

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#4661] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

项目模板新建后没有带解决方案文件 BootstrapBlazorApp.sln

### Steps To Reproduce

1. 安装文档项目模板 https://www.blazor.zone/template
2. dotnet new install Bootstrap.Blazor.Templates::9.0.0
3. dotnet new bbapp
4. 生成的文件夹里面没有带解决方案文件 BootstrapBlazorApp.sln , 小白会带来一些困扰.

### Exceptions (if any)

项目模板新建后没有带解决方案文件 BootstrapBlazorApp.sln

### .NET Version

9

### Anything else?

项目模板新建后没有带解决方案文件 BootstrapBlazorApp.sln

**Comment by bb-auto** [comment]
@densen2014 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply]
@densen2014 已发布 `9.0.1` 版本使用下方命令更新

`dotnet new install Bootstrap.Blazor.Templates::*`
